src/core2/graph.js

```diff
@@ -145,8 +199,18 @@ export class Graph {
   }
 
   equals(that: Graph): boolean {
```

Comment:
We should document that this tests for structural equality, and does not check the `plugins`.